internal/handler/handler.go

```diff
@@ -70,13 +72,30 @@ func (h *Hand) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.UpdateEmployee(w, r)
 		return
 	default:
-		fmt.Println("errors: not found")
 		http.NotFound(w, r)
 		return
 	}
 }
 
+func (h *Hand) getInfo(r *http.Request) error {
```

Comment:
`getInfo()` is too general name. The name of function should reflect what it does. From the function name developer should understand what a function does without opening code of the  function.